Apps/Src/Voltage.c

```diff
@@ -53,19 +53,19 @@ void Voltage_Init(cell_asic *boards){
     LTC6811_Init(Minions);
     
     //take control of mutex
-      RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
+    RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
     // Write Configuration Register
     LTC6811_wrcfg(NUM_MINIONS, Minions);
     //release mutex
-      RTOS_BPS_MutexPost(&MinionsASIC_Mutex, OS_OPT_POST_NONE);
+    RTOS_BPS_MutexPost(&MinionsASIC_Mutex, OS_OPT_POST_NONE);
 
-    // Read Configuration Register
-    // take control of mutex
-      RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
     wakeup_sleep(NUM_MINIONS);
```

Comment:
do we need this wakeup_sleep() ? its already called in this function at the beginning

Reply:
if a thread interrupts in that time, then we will wait 10ms and the ltc6811 will have gone back to sleep